test/locators.spec.ts

```diff
@@ -28,6 +29,11 @@ describe('Enter webpage', () => {
           'WebElement Commands'],
       });
     });
+    it('then should have a file uploaded', async () => {
+      const personalInformationPage: PersonalInformationPage = new PersonalInformationPage();
+
+      await expect(personalInformationPage.getUploadInputLength()).toBe('C:\\fakepath\\image_testing.jpg');
```

Comment:
in this case, you can change the assertion toContain and just use the "image_testing.jpg"